templates/subscriptionsdefinitionssubscriptions.json

```diff
@@ -0,0 +1,40 @@
+{
+  "_id": "{{@ system @}}-{{@ datatype @}}-all",
+  "add_namespaces": false,
+  "source": {
+    "dataset": "{{@ system @}}-subscriptionsdefinitions-collect",
+    "type": "dataset"
+  },
+  "transform": [
+    {
+      "rules": {
+        "default": [
+          [
+            "copy",
+            "*"
+          ],
+          [
+            "add",
+            "::properties",
```

Comment:
Use "namespaced_identfiers": false in all the connector pipes, that way you avoid adding `::`